application/models/file.go

```diff
@@ -206,3 +206,76 @@ func (f *File) Create() error {
 func (f *File) Update() error {
 	return update(f)
 }
+
+// DeleteUnlinked removes all files that are no longer linked to any database records
+func (f *Files) DeleteUnlinked() error {
+	var files Files
+	if err := DB.Select("id").All(&files); err != nil {
+		return err
+	}
+
+	toDelete := make(map[int]bool, len(files))
```

Comment:
Does using the len() for creating a map do anything useful.

Reply:
I had the same question, and I think it allocates enough memory for all of the pointers.